tests/html/measurements.html

```diff
@@ -0,0 +1,984 @@
+  <script type="text/javascript">
+//<![CDATA[
+var AUTH_TOKEN = "IkhuvwNdzpZPe2PnlQzR2emHWhYm4VN55kX5yffERbk=";
+//]]>
+</script>
+
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+
+  <title>Free Calorie Counter, Diet &amp; Exercise Journal | MyFitnessPal.com</title>
+
+  <meta name="description" content="Free online calorie counter and diet plan. Lose weight by tracking your caloric intake quickly and easily.  Find nutrition facts for over 2,000,000 foods." />
+  <meta name="keywords" content="calorie counter, free diet journal, weight loss program, nutrition facts, online calorie counter, free diet plan, weight loss online, free calorie counter" />
+  <meta content="authenticity_token" name="csrf-param" />
+<meta content="IkhuvwNdzpZPe2PnlQzR2emHWhYm4VN55kX5yffERbk=" name="csrf-token" />
+  <meta name="verify-v1" content="jKS1fjb9pK13luol4O+VFWO0OwEYaK4au88j0aRpQTM=" />
+
+
+  <script type="text/javascript">
+    document.documentElement.className += " js-enabled";
+  </script>
+
+    <link href="http://d34yn14tavczy0.cloudfront.net/assets/sass/application-ee21d18aff36a7fb86edf7018aeb9003.css" media="screen" rel="stylesheet" type="text/css" />
+
+
+  <link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet">
+  <!--[if IE 7]>
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome-ie7.min.css">
+  <![endif]-->
+  <link href="http://d34yn14tavczy0.cloudfront.net/stylesheets/font-mfizz/font-mfizz.css" media="all" rel="stylesheet" type="text/css" />
+
+  
+
+
+  <script>
+var MFP = (function (parent) {
+  parent.URL = (function () {
+    var me = {};
+    me.currentLocale = "en";
+    me.baseUrl = "http://www.myfitnesspal.com/";
+    me.httpsBaseUrl = "https://www.myfitnesspal.com/";
+    me.topLevelDomain = "com";
+    me.defaultLocale = "en";
+    me.getUrlPath = function(path) {
+      //if (me.topLevelDomain == me.currentLocale) {
+      if (me.defaultLocale == me.currentLocale) {
+        return (path[0] == '/' ? path.substring(1) : path);
+      } else {
+        return '/' + me.currentLocale + path;
+      }
+    }
+    me.getUrlAbsolute = function(path) {
+      return me.baseUrl + (path[0] == '/' ? path.substring(1) : path);
+    }
+    me.getHttpsUrlAbsolute = function(path) {
+      if (false) {
+        return me.getUrlAbsolute(path);
+      } else {
+        return me.httpsBaseUrl + (path[0] == '/' ? path.substring(1) : path);
+      }
+    }
+    return me;
+  }());
+  return parent;
+})(MFP || {});
+</script>
+
+  <script>
+var MFP = (function (parent) {
+  parent.DEBUG = false;
+  parent.DEVELOPMENT = false;
+
+  return parent;
+})(MFP || {});
+</script>
+
+    <script>
+var MFP = (function (parent) {
+  parent.User = (function () {
+    var me = {};
+    me.MFP_API_HOST = 'https://api.myfitnesspal.com/v2';
+    me.MFP_CLIENT_ID = 'mfp-main-js';
+    me.api_user_id = '195472177090541';
```

Comment:
You might want to push up a slightly-modified version of this removing any personally-identifying information like this user ID (as well as the below OAuth access_token)